### PR TITLE
Update dlp-policy-reference.md

### PR DESCRIPTION
**Pull request by Pulkit-Sachan**

### CODE REVIEW

File: microsoft-365/compliance/dlp-policy-reference.md [hunk 1]
```diff
@@ -880,7 +880,8 @@ This table shows the DLP blocking and notification behavior for policies that ar
...
+|- **Any one condition to be present in rule ** |  - **Restrict access or encrypt the content in Microsoft 365 locations** is selected </br>- **Block users from receiving email or accessing shared SharePoint, OneDrive, and Teams files** is selected </br>- **Block everyone** is selected        | - **User notifications** set to **On** </br>- **Notify users in Office 365 service with a policy tip** is selected </br>- **Notify the user who sent, shared, or last modified the content** is selected         | - **Send an alert to admins when a rule match occurs** set to **On** </br>- **Send alert every time an activity matches the rule** is selected </br>- **Use email incident reports to notify you when a policy match occurs** set to **On**        |Notifications are sent when a file is shared with an external user and an external user access that file.         |
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/dlp-policy-reference.md:883]:
```suggestion
|- **Any one condition to be present in rule** |  - **Restrict access or encrypt the content in Microsoft 365 locations** is selected </br>- **Block users from receiving email or accessing shared SharePoint, OneDrive, and Teams files** is selected </br>- **Block everyone** is selected        | - **User notifications** set to **On** </br>- **Notify users in Office 365 service with a policy tip** is selected </br>- **Notify the user who sent, shared, or last modified the content** is selected         | - **Send an alert to admins when a rule match occurs** set to **On** </br>- **Send alert every time an activity matches the rule** is selected </br>- **Use email incident reports to notify you when a policy match occurs** set to **On**        |Notifications are sent when a file is shared with an external user and an external user access that file.         |
```

File: microsoft-365/compliance/dlp-policy-reference.md [hunk 2]
```diff
@@ -880,7 +880,8 @@ This table shows the DLP blocking and notification behavior for policies that ar
...
+|- **Any one condition to be present in rule**    |- **Restrict access or encrypt the content in Microsoft 365 locations** is selected </br>- **Block only people who were given access to the content through the "Anyone with the link" option** is selected.         |  - **User notifications** set to **On** </br>- **Notify users in Office 365 service with a policy tip** is selected.  </br>- **Notify the user who sent, shared, or last modified the content** is selected     |- **Send an alert to admins when a rule match occurs** set to **On**   </br>- **Send alert every time an activity matches the rule** is selected </br>- **Use email incident reports to notify you when a policy match occurs** set to **On**       |Notifications are sent as soon as a file is uploaded         |
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/dlp-policy-reference.md:884]:
```suggestion
|- **Any one condition to be present in rule**    |- **Restrict access or encrypt the content in Microsoft 365 locations** is selected </br>- **Block only people who were given access to the content through the "Anyone with the link" option** is selected.         |  - **User notifications** set to **On** </br>- **Notify users in Office 365 service with a policy tip** is selected.  </br>- **Notify the user who sent, shared, or last modified the content** is selected     |- **Send an alert to admins when a rule match occurs** set to **On**   </br>- **Send alert every time an activity matches the rule** is selected </br>- **Use email incident reports to notify you when a policy match occurs** set to **On**       |Notifications are sent as soon as a file is uploaded.         |
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @Pulkit-Sachan, kindly commit the minor suggested changes. Many thanks.

**Comment by yogkumgit** [comment]
@Pulkit-Sachan Thanks for your feedback! New issues and pull requests for this content are closed while we update this content to support upcoming feature improvements.

**Pull request closed without merging**